net/tor/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tor
-PKG_VERSION:=0.4.7.8
+PKG_VERSION:=0.4.7.9
```

Comment:
I wonder, why not update it rather to 0.4.7.10?

Reply:
Thanks. :) Updated.